statick_tool/statick.py

```diff
@@ -64,6 +62,28 @@ def __init__(self, user_paths: List[str]) -> None:
         self.config = None  # type: Optional[Config]
         self.exceptions = None  # type: Optional[Exceptions]
 
+    @staticmethod
+    def set_logging_level(args: argparse.Namespace) -> None:
+        """Set the logging level to use for output.
+
+        Valid levels are: DEBUG, INFO, WARNING, ERROR, CRITICAL. Specifying the level is
+        case-insensitive (both upper-case and lower-case are allowed).
+        """
+        valid_levels = ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
+        log_level = args.log_level.upper()
+        if log_level not in valid_levels:
+            log_level = "WARNING"
+        args.log_level = log_level
+        logging.basicConfig(level=log_level)
+        logging.root.setLevel(log_level)
+        logging.info("Log level set to %s", args.log_level.upper())
+
+        if args.show_tool_output:
+            logging.warning(
```

Comment:
do we want to automatically set the log level to debug if this happens? at least until v0.6?

Reply:
Good idea. I added that.